src/components/visualizer.tsx

```diff
@@ -71,3 +97,12 @@ export default function Visualizer({ inVisualizer, current, goBack }: Visualizer
     </div>
   );
 }
+
+function delay(millis: number): Promise<void> {
+  return new Promise((resolve, reject) => {
+    setTimeout(
+      () => { resolve(); },
+      millis
+    );
+  });
+}
```

Comment:
This could/should probably be moved to `src/util/utils.ts`

Reply:
Done